wd_helper.au3

```diff
@@ -1679,7 +1682,7 @@ Func _WD_GetBrowserPID($iWebDriverPID, $sBrowserName = '')
 			If Not $iBrowserPID Then $iErr = $_WD_ERROR_NoMatch ; BrowserExe related to requested BrowserName was not matched in the webdriver child process list
 		EndIf
 	EndIf
-	Return SetError(__WD_Error($sFuncName, $iErr), 0, $iBrowserPID)
+	Return SetError(__WD_Error($sFuncName, $iErr, $iExt), $iExt, $iBrowserPID)
```

Comment:
third parameter for `__WD_Error` is `$sMessage` not `$iExt`

This led me to these changes: [$iExt + $sMessage usage](https://github.com/Danp2/au3WebDriver/pull/324/commits/5030e2a9c1adba6bc356a1dde7c3e127496ccac6)

Reply:
If you accept this recent commit ([$iExt + $sMessage usage](https://github.com/Danp2/au3WebDriver/pull/324/commits/5030e2a9c1adba6bc356a1dde7c3e127496ccac6)) then this entire PR is fine for me and is finally ready to merge.

Reply:
I see that you added a unique $iExt value for each exit point. With that in mind, can we eliminate some additional error codes (change them all to $_WD_ERROR_GeneralError)?

Reply:
Please take a look on:
[ProcessExists() and $iExt usage](https://github.com/Danp2/au3WebDriver/pull/324/commits/8c80c8e829f4342fe04bb03b6238f04b78c8441b)